src/popup/popup.js

```diff
@@ -1,21 +1,8 @@
 import './popup.scss';
+import Vue from 'vue';
```

Comment:
At first, I made the choice not to use any framework for the Popup because I was afraid it would took a little bit longer to load, but thanks to your PR, now I know that the difference is not noticeable :)